react/CircleButton/helpers.spec.js

```diff
@@ -0,0 +1,61 @@
+import { makeTypoColor, makeButtonShadow } from './helpers'
+
+const theme = {
+  palette: {
+    primary: { main: 'white' },
+    text: { disabled: 'lightGrey', primary: 'grey' },
+    border: { main: 'lightGrey' }
```

Comment:
nit: can we use a different color in this case?

Reply:
ah oui, c'est moins représentatif mais plus simple à vérifier dans le test effectivement